src/services/user/registration.js

```diff
@@ -152,6 +152,13 @@ const registerUser = function register(data, params, app) {
 		)
 		.then(() => {
 			const consentSkipCondition = Configuration.get('SKIP_CONDITIONS_CONSENT');
+			if (
+				(user.roles || []).filter((role) => {
+					return ['student', 'employee', 'expert', 'administrator', 'teacher'].includes(role);
```

Comment:
please put the list to a constant, so we can easily find it and change if necessary

Reply:
done